cmd/osv-scanner/__snapshots__/main_test.snap

```diff
@@ -1064,43 +1064,43 @@ Scanned <rootdir>/fixtures/locks-requirements/the_requirements_for_test.txt file
 
 ---
 
-[TestRun/requirements.txt_can_have_all_kinds_of_names - 2]
+[Test_run/requirements.txt_can_have_all_kinds_of_names - 2]
 
 ---
 
-[TestRun/verbosity_level_=_error - 1]
+[Test_run/verbosity_level_=_error - 1]
 No issues found
 
 ---
 
-[TestRun/verbosity_level_=_error - 2]
+[Test_run/verbosity_level_=_error - 2]
 
 ---
 
-[TestRun/verbosity_level_=_info - 1]
+[Test_run/verbosity_level_=_info - 1]
 Scanning dir ./fixtures/locks-many/composer.lock
 Scanned <rootdir>/fixtures/locks-many/composer.lock file and found 1 package
 Loaded filter from: <rootdir>/fixtures/locks-many/osv-scanner.toml
 No issues found
 
 ---
 
-[TestRun/verbosity_level_=_info - 2]
+[Test_run/verbosity_level_=_info - 2]
 
 ---
 
-[TestRun/version - 1]
+[Test_run/version - 1]
 osv-scanner version: 2.0.0-beta2
 commit: n/a
 built at: n/a
 
 ---
 
-[TestRun/version - 2]
+[Test_run/version - 2]
 
 ---
 
-[TestRunCallAnalysis/Run_with_govulncheck - 1]
+[Test_runCallAnalysis/Run_with_govulncheck - 1]
```

Comment:
I'm a bit confused by the naming here. I thought we were converting all test functions to the `Test_run_<OriginalFunctionName>` format. Why isn't this one named `Test_run_CallAnalysis`?

Reply:
btw, why is `run` in lowercase?

Reply:
> Why isn't this one named Test_run_CallAnalysis?

I believe it should be, I've just mucked something up (things are a little hard to pin down right now between all the recent CI flake and the massive output from `osv-scalibr` 😅)

> btw, why is run in lowercase?

because `run` is a private method - also, just to be clear, if it was public then the name would be `TestRun`; the reason we add the underscore is that Go requires camel case names + prefixed with `Test`, and its possible to have a public and private thing with the "same" name i.e.

```
func Run() { } // -> TestRun
func run() { }  // -> Test_run
```